APMrover2/system.cpp

```diff
@@ -229,6 +229,7 @@ bool Rover::set_mode(Mode &new_mode, mode_reason_t reason)
 #if FRSKY_TELEM_ENABLED == ENABLED
     frsky_telemetry.update_control_mode(control_mode->mode_number());
 #endif
+    camera.update_control_mode(control_mode->mode_number());
```

Comment:
Missing the macros for CAMERA being enabled (applies to all vehicles).

Reply:
not sure if you mean to tell me that APMrover2 does not use AP_Camera ? - or what ?  I have it working, so "all vehicles" is unclear to me.

Reply:
Example: https://github.com/ArduPilot/ardupilot/blob/master/APMrover2/APMrover2.cpp#L207

And this applies to all vehicles - you have to check if the macro is exactly the same in all vehicles.

Reply:
Thank you, now I get it, in case it was compiled without camera. Will fix.